plugins/base/src/main/kotlin/parsers/MarkdownParser.kt

```diff
@@ -37,7 +53,7 @@ open class MarkdownParser(
         return visitNode(markdownAstRoot)
     }
 
-    override fun preparse(text: String) = text
+    override fun preparse(text: String) = text.replace("\r\n", "\n").replace("\r", "\n")
```

Comment:
I have a bad feeling about that. What would happen if somebody would add `\r\n` in their documentation? Or any other escaped version of those characters?

Reply:
I assume (unless you're doing some kind of preprocessing somewhere else) that they won't be unescaped, i.e. a "\n" in the markdown document will be parsed as `['\', 'n']` i.e. `"\\n"`, not `['\n']`.  I'll add this to the tests.